setup.py

```diff
@@ -1,50 +1,45 @@
-import os
-from setuptools import setup, find_packages
+from distutils.core import setup
+import setuptools
+import sys
+from os.path import join
 
-here = os.path.abspath(os.path.dirname(__file__))
-
-requires = [
-    'dropbox',
-    'boxsdk',
-    'LAMP-core',
-    'paramiko',
-    'pyyaml',
-    'mano',
-    'yaxil',
-    'six',
-    'pytz',
-    'pandas',
-    'jsonpath_ng',
-    'dpanonymize@git+https://github.com/AMP-SCZ/dpanonymize.git'
-]
-
-test_requirements = [
-    'pytest',
-    'vcrpy'
-]
-
-about = dict()
-with open(os.path.join(here, 'lochness', '__version__.py'), 'r') as f:
-    exec(f.read(), about)
+with open("README.md", "r") as fh:
+    long_description = fh.read()
 
 setup(
-    name=about['__title__'],
-    version=about['__version__'],
-    description=about['__description__'],
-    author=about['__author__'],
-    author_email=about['__author_email__'],
-    url=about['__url__'],
-    packages=find_packages(),
-    package_data={
-        '': ['*.yaml', '*.html']
-    },
-    scripts=[
-        'scripts/sync.py',
-        'scripts/phoenix_generator.py',
-        'scripts/listen_to_redcap.py',
-        'scripts/lochness_check_config.py',
-        'scripts/lochness_create_template.py',
+    name='ampscz-lochness',
+    version='v0.1.2',
+    description='AMP-SCZ lochness',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author='DPACC',
+    author_email='kevincho@bwh.harvard.edu',
+    url='https://github.com/AMP-SCZ/lochness',
+    download_url='https://github.com/AMP-SCZ/lochness/archive/refs/tags/v0.1.0.zip',
```

Comment:
Shouldn't it be `v.0.1.2.zip` according to `version` above?

Reply:
759defd